pom.xml

```diff
@@ -64,10 +64,8 @@
 
     <opentelemetry.version>1.43.0</opentelemetry.version>
     <!-- Alpha jar! -->
-    <opentelemetry-proto.version>1.3.2-alpha</opentelemetry-proto.version>
+    <opentelemetry-proto.version>1.4.0-alpha</opentelemetry-proto.version>
     <opentelemetry-semconv.version>1.28.0-alpha</opentelemetry-semconv.version>
```

Comment:
Please, since you are on it:

```suggestion
    <opentelemetry-semconv.version>1.29.0-alpha</opentelemetry-semconv.version>
```

Reply:
I’ll remove the dependency on otel semconv jar in the next pull request.

Reply:
My sincere apologies, for some reasons I was sure 1.29.0-alpha is out, but 1.28.0-alpha is indeed the latest one, sorry about that

Reply:
later, we can probably move it to a test dep only, to make sure that attribute names are correct and break tests if they drift?

Reply:
No worries!